fairseq/tasks/multilingual_translation.py

```diff
@@ -74,9 +74,9 @@ def add_args(parser):
                             help='source language (only needed for inference)')
         parser.add_argument('-t', '--target-lang', default=None, metavar='TARGET',
                             help='target language (only needed for inference)')
-        parser.add_argument('--left-pad-source', default='True', type=str, metavar='BOOL',
+        parser.add_argument('--left-pad-source', default=True, type=str, metavar='BOOL',
                             help='pad the source on the left (default: True)')
-        parser.add_argument('--left-pad-target', default='False', type=str, metavar='BOOL',
+        parser.add_argument('--left-pad-target', default=False, type=str, metavar='BOOL',
```

Comment:
Can you clarify why this change is needed? I believe we cast them in lines 122-123 below (similar to `tasks/translation.py`)